README.md

```diff
@@ -31,8 +31,8 @@ Paloma.js can be used in Node.js, as well as inside the browser.
 import { LCDClient, MsgSend, MnemonicKey } from '@palomachain/paloma.js';
 
 const lcd = new LCDClient({
-  URL: 'http://rbox.palomachain.com:1317',
-  chainID: 'paloma-testnet-6',
+  URL: 'http://testnet.palomaswap.com:1317',
+  chainID: 'paloma-testnet-9',
```

Comment:
I think a better way to do this is to remove `chainID` here and just start setting
```
in .paloma/config/client.toml

chain-id = "paloma-testnet-9"
```
because this is getting a little rediculous.